foam_internalwrapper/foam_internalwrapper.py

```diff
@@ -0,0 +1,260 @@
+""" foam_internalwrapper module
+
+Wrapper module for OpenFOAM
+
+"""
+from simphony.core.cuba import CUBA
+from simphony.core.data_container import DataContainer
+from simphony.cuds.abc_modeling_engine import ABCModelingEngine
+from PyFoam.Execution.ConvergenceRunner import ConvergenceRunner
+from PyFoam.LogAnalysis.BoundingLogAnalyzer import BoundingLogAnalyzer
+from foam_files import FoamFiles
+from cuba_extension import CUBAExt
+from foam_mesh import FoamMesh
+import simphonyfoaminterfaceII as foamface
+import os
+
+
+class FoamInternalWrapper(ABCModelingEngine):
+    """ Wrapper to OpenFOAM
+
+    """
+
+    def __init__(self):
+
+        self._meshes = {}
+        self.CM = DataContainer()
+        self.BC = DataContainer()
+        self.SP = DataContainer()
+        #: to be able to use CUBAExt keywords, which are not in accepted
+        #  CUBA keywords these extensions to CM and SP is used
+        self.CM_extensions = {}
+        self.SP_extensions = {}
+
+    def run(self):
+        """ run OpenFoam based on CM, BC and SP data
+
+        Returns
+        -------
+        lastTime : lastTime
```

Comment:
@khiltunen and @jmarcelogimenez   an issue should be created as both the FILE-IO and INTERNAL wrappers to adress that the run method is incorrectly returning something